### ISSUE TITLE
Add WinidiCSS.

### ISSUE THREAD

**Issue by shadear-bakr** [troxler/awesome-css-frameworks#74]
Windi CSS was a compiler for tailwind, but since the tailwind JIT release they are no longer a compiler.

https://windicss.org/
> Next generation utility-first CSS framework.

https://github.com/windicss/windicss

**Comment by troxler** [comment]
Thanks for the suggestion.

Honestly, I don't understand the appeal of this framework at all. It seems to be Tailwind with additional syntax to replace actual CSS with their own version. Why would I use "value auto-infer" instead of just CSS? What is the advantage of `.btn { @apply hover:bg-blue-700; }` over `.btn { &:hover { @apply bg-blue-700 } }`? Why would you use a shortcut instead of `@apply`? And in general, I think it is very bad that these utility-based frameworks do not enforce some sane focus stylings by default. Obviously they are often forgotten and even Windi's website itself is not usable with the keyboard because of that.

Are you involved in that project? Maybe you could share your insights either as a developer of Windi or as a user.

**Comment by shadear-bakr** [comment]
Thanks for the elaboration, I honestly don't know how it is implemented and why it is implemented that way.
I was introduced to tailwindcss when I was doing a JS course and I followed the project and then I found windi was a faster option, but at that moment I didn't really cared about it.
Until tailwind implemented JIT and didn't mention windicss team efforts (at the launching time) so I stopped using tailwind. 
Sorry if I can't help with that, maybe create an issue on there Github and wait for official answers?
